ios/Demo-iOS/Sources/EditorView.swift

```diff
@@ -3,33 +3,34 @@ import GutenbergKit
 
 struct EditorView: View {
     private let configuration: EditorConfiguration
+    @Environment(\.dismiss) private var dismiss
 
     init(configuration: EditorConfiguration) {
         self.configuration = configuration
     }
 
     var body: some View {
         _EditorView(configuration: configuration)
+            .navigationBarBackButtonHidden(true)
             .toolbar {
-                ToolbarItemGroup(placement: .topBarLeading) {
-                    Button(action: {}, label: {
+                ToolbarItem(placement: .topBarLeading) {
+                    Button(action: {
+                        dismiss()
+                    }, label: {
                         Image(systemName: "xmark")
                     })
                 }
+
                 ToolbarItemGroup(placement: .topBarTrailing) {
                     Button(action: {}, label: {
                         Image(systemName: "arrow.uturn.backward")
```

Comment:
I would be nice to connect these buttons to the editor.

Reply:
Agreed!

Reply:
Addressed in https://github.com/wordpress-mobile/GutenbergKit/pull/187.